packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -370,4 +379,6 @@ class VideoPlayerOptions {
   /// Note: This option will be silently ignored in the web platform (there is
   /// currently no way to implement this feature in this platform).
   final bool mixWithOthers;
+
+  final bool bypassAudioSetup;
```

Comment:
It would be better to phrase this as the inverse, and default it to true, especially since we'd like to change the default later in a breaking change and "bypassAudioSetup: false" would be a weird way of expressing something that won't be default.

How about `setGlobalAudioMode` or something along those lines?